dialog.txt

```diff
@@ -36,70 +36,70 @@ Errore di sistema!!!#|
 *È stato terribile...#|
 *Zombi!#|
 *Ah-ah...# Diventerai uno# Zombiiii!#|
-*Di certo qualcuno# ha il controllo# dei morti.#|
-*Sei stato bravo,# giovine.#|
+*Qualcuno deve# avere il controllo# dei morti.#|
+*Sei stato bravo,# giovanotto.#|
 *Gli zombi potrebbero# essere in qualunque# parte del paese.#|
-*Quel Canarino# Pulcino viene dal# Villaggio Canarino?#|
+*Quel pulcino di# canarino viene dal# Villaggio Canarino?#|

```

Comment:
Cambierei in "Pulcino di Canarino", visto che i nomi degli oggetti hanno le maiuscole.